### DISCUSSION TITLE
Connect Postal to another mail gateway

### DISCUSSION THREAD

**Discussion by ashcaan** [postalserver/postal#2546] [Help with using Postal] [answered]
Hello,

We are currently exploring the potential of utilizing Postal as an outgoing mail gateway for our diverse client base, which includes users employing different mail servers such as Exim, Postfix and etc.

While delving into the documentation, we couldn't find concrete information regarding the integration capabilities of Postal with external mail gateways like Halon, specifically for processing and sending outgoing messages.

Therefore, we kindly request your assistance in clarifying whether it is possible to establish a connection between Postal and a mail gateway like Halon, enabling Postal to effectively pass mail to Halon to handle outgoing emails.

We greatly appreciate any insights, recommendations, or guidance you can provide on this matter.

Thank you in advance for your time and support!

**Comment by marabzadeh86** [comment]
Hi,
Yeah I was reading through docs and browsing different discussion here searching for the same information.
Is there a "smart host" option that we can use to route all email to an upstream mail server/gateway?

**Comment by slobodangacesa** [comment]
You can use relay (smart host) but without auth: https://github.com/postalserver/postal/blob/main/config/postal.defaults.yml#L79-L83

**Comment by willpower232** [comment]
Postal isn't designed to be an outgoing email gateway, it is mostly intended to enable software to send emails.

I think Postfix would make a better outgoing email gateway than Postal.

If you wanted to use Postal, you would have to configure each route through Postal and there would be a lot of configuration required which would be annoying to maintain long term.

**Comment by adamcooke** [comment, marked as answer]
You can configure SMTP relays which will send all outbound emails to that host rather than to the actual recipients mail server. This can be used to achieve this result.
